lib/platform/windows.c

```diff
@@ -561,4 +561,11 @@ struct switchtec_dev *switchtec_open_i2c_by_adapter(int adapter, int i2c_addr)
 	return NULL;
 }
 
+struct switchtec_dev *switchtec_open_uart(const char *path)
+{
+	errno = ENOTSUP;
+	return NULL;
+}
+
+
```

Comment:
Nit: Avoid extra lines in a few places.... (kernel style)

Reply:
fixed